.github/workflows/build.yml

```diff
@@ -0,0 +1,57 @@
+name: Build
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    branches: [ master, develop ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+        git fetch --prune --unshallow
+    - name: Dump GitHub context
+      env:
+        GITHUB_CONTEXT: ${{ toJson(github) }}
+      run: echo "$GITHUB_CONTEXT"
+    - name: setup-docker
+      uses: docker-practice/actions-setup-docker@0.0.1
+    # https://github.community/t/worker-running-out-of-disk-space/18245
+    - name: Patch Docker Daemon data-root
+      run: |
+        DOCKER_DATA_ROOT='/mnt/var/lib/docker'
+        DOCKER_DAEMON_JSON='/etc/docker/daemon.json'
+        sudo mkdir -p "${DOCKER_DATA_ROOT}"
+        jq --arg dataroot "${DOCKER_DATA_ROOT}" '. + {"data-root": $dataroot}' "${DOCKER_DAEMON_JSON}" > "/tmp/docker.json.tmp"
+        sudo mv "/tmp/docker.json.tmp" "${DOCKER_DAEMON_JSON}"
+        sudo systemctl restart docker
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '^1.14.3' # The Go version to download (if necessary) and use.
+    - name: Login to DockerHub Registry
+      if: github.ref == 'refs/heads/master' && github.event_name == 'push'
+      run: echo ${{ secrets.QUAY_PASSWORD }} | sudo -E docker login -u ${{ secrets.QUAY_USERNAME }} --password-stdin quay.io
+
+    - name: Set Push options
+      if: github.ref == 'refs/heads/master' && github.event_name == 'push'
+      run: echo "BUILD_ARGS=--push --only-target-package --pull --image-repository quay.io/mocaccinoos/stage3-amd64-cache" >> $GITHUB_ENV
+    - name: Install and Build 🔧 # This example project is built using npm and outputs the result to the 'build' folder. Replace with the commands required to build your project, or remove this step entirely if your site is pre-built.
+      env:
+        PREVIOUS: ${{ github.event.before }}
+      run: |
+        curl https://get.mocaccino.org/luet/get_luet_root.sh | sudo sh
+        luet build --all --destination=${PWD}/build --tree=${PWD}
+        luet create-repo --name "mocaccino-stage3" --output ${PWD}/build --packages ${PWD}/build --tree ${PWD}
```

Comment:
```suggestion
        luet create-repo --name "mocaccino-stage3" --output ${PWD}/build --packages ${PWD}/build --tree ${PWD}/packages
```

Reply:
we do this for one reason, during `create-repo`, we snapshot the compilation specs, so if you specify $PWD, it would snapshot also the `build` content